esp/protocols/http/CMakeLists.txt

```diff
@@ -76,6 +76,7 @@ target_link_libraries(
     ${XALAN_LIBRARIES}
     ${XERCES_LIBRARIES}
     jlib
+	dalibase
```

Comment:
This change looks unrelated - it should be in a separate pull request and a separate Jira.